src/metabase/models/cache_config.clj

```diff
@@ -5,6 +5,8 @@
    [methodical.core :as methodical]
    [toucan2.core :as t2]))
 
+(def CacheConfig "Cache configuration" :model/CacheConfig)
```

Comment:
Got into "it wasn't imported" situation again. 🙄

Reply:
how? if `metabase.models` is loaded then this ns should be loaded no?

Reply:
Yes, but when you're not importing `CacheConfig` to use (just literally type in `:model/CacheConfig`), then there is no guarantee that `metabase.models` is loaded. 😁

Reply:
We can bring that up at backend guild sync at some point I think